presto-main/src/test/java/com/facebook/presto/operator/scalar/TestBitwiseFunctions.java

```diff
@@ -90,4 +92,48 @@ public void testBitwiseXor()
         assertFunction("bitwise_xor(-4, 12)", BIGINT, -4L ^ 12L);
         assertFunction("bitwise_xor(60, 21)", BIGINT, 60L ^ 21L);
     }
+
+    @Test
+    public void testBitwiseSll()
+    {
+        assertFunction("bitwise_sll(7, 2, 4)", BIGINT, 12L);
```

Comment:
Maybe we can spell out sll, srl, and sra for users to better understand the semantics.